### DISCUSSION TITLE
重启上亿数据量的 milvus 集群 milvus proxy 和 milvus coord 两个结点一直重启

### DISCUSSION THREAD

**Discussion by 1433428225** [milvus-io/milvus#28515] [Q&A and General discussion]
[ERROR] [meta/coordinator_broker.go:164] ["get recovery info failed"] [collectionID=442581645051993835] [partitionIDs="[442581645051993836]"] [error="code: CollectionNotExists, reason: can't find collection: 442581645051993835"] [stack="[github.com/milvus-io/milvus/internal/querycoordv2/meta.(*CoordinatorBroker).GetRecoveryInfoV2\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/coordinator_broker.go:164\ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).PullNextTargetV2.func1\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:207\ngithub.com/milvus-io/milvus/internal/util/retry.Do\n\t/go/src/github.com/milvus-io/milvus/internal/util/retry/retry.go:42\ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).PullNextTargetV2\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:229\ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).updateCollectionNextTarget\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:125\ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).UpdateCollectionNextTarget\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:117\ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recoverCollectionTargets.func1\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:589\ngithub.com/milvus-io/milvus/internal/util/retry.Do\n\t/go/src/github.com/milvus-io/milvus/internal/util/retry/retry.go:42\ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recoverCollectionTargets\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:588\ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recover.func1\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:573\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/go/pkg/mod/golang.org/x/sync@v0.1.0/errgroup/errgroup.go:75](http://github.com/milvus-io/milvus/internal/querycoordv2/meta.(*CoordinatorBroker).GetRecoveryInfoV2/n/t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/coordinator_broker.go:164/ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).PullNextTargetV2.func1/n/t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:207/ngithub.com/milvus-io/milvus/internal/util/retry.Do/n/t/go/src/github.com/milvus-io/milvus/internal/util/retry/retry.go:42/ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).PullNextTargetV2/n/t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:229/ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).updateCollectionNextTarget/n/t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:125/ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).UpdateCollectionNextTarget/n/t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:117/ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recoverCollectionTargets.func1/n/t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:589/ngithub.com/milvus-io/milvus/internal/util/retry.Do/n/t/go/src/github.com/milvus-io/milvus/internal/util/retry/retry.go:42/ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recoverCollectionTargets/n/t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:588/ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recover.func1/n/t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:573/ngolang.org/x/sync/errgroup.(*Group).Go.func1/n/t/go/pkg/mod/golang.org/x/sync@v0.1.0/errgroup/errgroup.go:75)"]

**Comment by 1433428225** [comment]
<img width="769" alt="image" src="https://github.com/milvus-io/milvus/assets/34463186/93c4bf8f-f24a-4975-8538-cf64839e0f6d">
![Uploading milvus.jpeg…]()

**Comment by xiaofan-luan** [comment]
@1433428225 
could you offer full logs for all coordinators?
